runtime/compiler/runtime/MetaDataDebug.cpp

```diff
@@ -553,35 +553,38 @@ extern "C" void jitBytecodePrintFunction(void *userData, char *format, ...)
 void
 TR_Debug::printByteCodeStack(int32_t parentStackIndex, uint16_t byteCodeIndex, char * indent)
    {
-      if (!_comp->fej9()->isAOT_DEPRECATED_DO_NOT_USE())
+#if defined(JITSERVER_SUPPORT)
```

Comment:
This means that when we are going to make this define the default, this function will stop working. We should check that we are not at the server. I see _comp being used below so this should be easy.